File: python-venv/find_the_vowel/kata_function/find_the_vowel.py
import re

def vowel(string):
    output = []

    for i in range(0, len(string), 1):
        upper = string[i].upper()
        if len(re.findall("[AEIOUY]", upper)) > 0:
            output.append(i + 1)

    return output

#https://www.codewars.com/kata/5680781b6b7c2be860000036/train/python

File: find-the-vowel/find-the-vowel.js
const vowelIndices = (string) => {
    const output = [];
    const vowels = "AEIOUY";

    string.split("").forEach((element, index) => {
        if(vowels.includes(element.toUpperCase())){
            output.push(index + 1);
        };
    });

    return output;
};

module.exports = { vowelIndices };


//https://www.codewars.com/kata/5680781b6b7c2be860000036/train/javascript